users.js

```diff
@@ -1086,16 +1086,18 @@ class User {
 		this.inRooms.clear();
 	}
 	getAlts(includeTrusted, forPunishment) {
-		return this.getAltUsers(includeTrusted, forPunishment).map(user => user.getLastName());
+		return this.getAltUsers(includeTrusted, forPunishment).filter(user => user.named).map(user => user.getLastName());
```

Comment:
Then it falls back to `Guest [...]`, but this should be rare enough not to really be a problem.